atomic_reactor/util.py

```diff
@@ -1942,7 +1942,7 @@ def terminal_key_paths(obj: dict) -> Iterator[Sequence]:
 
     From `{'a': {'b': {'c': 1, 'd': 2}}}` yields `('a', 'b', 'c')`, `('a', 'b', 'd')`
     """
-    stack = [
+    stack: List[Tuple[Any, Tuple[str, ...]]] = [
```

Comment:
It would be good for the path type to match the return type of this function (`Tuple[str, ...]` vs. `Sequence`)

Reply:
I can update function definition to `-> Tuple[str, ...]` not opposite. I tried and it doesn't like + operator. But given function indeed returns Tuples, func annotation should be updated